timemachine/ff/handlers/utils.py

```diff
@@ -67,3 +73,57 @@ def match_smirks(mol, smirks):
         matches.append(tuple(mas))
 
     return matches
+
+
+def convert_to_oe(mol):
+    """Convert an ROMol into an OEMol"""
+
+    # imported here for optional dependency
+    from openeye import oechem
+
+    mb = Chem.MolToMolBlock(mol)
+    ims = oechem.oemolistream()
+    ims.SetFormat(oechem.OEFormat_SDF)
+    ims.openstring(mb)
+
+    for buf_mol in ims.GetOEMols():
+        oemol = oechem.OEMol(buf_mol)
+    ims.close()
+    return oemol
+
+
+def get_symmetry_classes(rdmol: Chem.Mol) -> NDArray:
+    """Convenient wrapper for [atom.GetSymmetryClass() for atom in oemol]
+
+    (Accepts rdmol instead of oemol, and renumbers classes to be contiguous from 0 to n_classes.)
+    """
+
+    # imported here for optional dependency
+    from openeye import oechem
+
+    oemol = convert_to_oe(rdmol)
+    oechem.OEPerceiveSymmetry(oemol)
+    symmetry_classes = np.array([atom.GetSymmetryClass() for atom in oemol.GetAtoms()])
+    n_classes = len(set(symmetry_classes))
+
+    # make contiguous from 0 to n_classes
+    idx_map = {old_idx: new_idx for (new_idx, old_idx) in enumerate(set(symmetry_classes))}
+    symmetry_classes = np.array([idx_map[old_idx] for old_idx in symmetry_classes])
+    assert set(symmetry_classes) == set(range(n_classes))
+
+    return symmetry_classes
+
+
+def symmetrize(per_particle_params, mol):
+    """Replace param[atom] with mean([param[a] for a in symmetry_class(atom)])"""
```

Comment:
nit: update to "Replace per_particle_params[atom] ..."

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/738/commits/511bc2b7b3c0b6cef6cc8d415a118614cfd2b58a